### PR TITLE
Initialization Testing

### PR DESCRIPTION
**Pull request by quantum9Innovation**
# Adds initialization testing to package
From PR #48, after PR #49 
___

## New Feature
Adds sanity and dependency checks to the `epispot/__init__.py` file and adds classifiers and other metadata to both `setup.py` and `setup-nightly.py`.
## Known Issues
No known issues.
## Code Breakdown
 - epispot/\_\_init\_\_.py
     - Sanity checks
     - Dependency checks
     - Increased package documentation
         - Ported from `README.md`
 - release.py
     - Updated to be compatible with new initialization file
 - setup.py, setup-nightly.py
     - Adds classifiers
     - Adds more URLs

### CODE REVIEW

**Comment by quantum9Innovation** [comment]
## Merging Guidelines
Test analysis

---

Build → passing ✅ 
Code Coverage → **not failing**\* (no change 🏳️ )
CodeQL → passing ✅ 
DeepSource → passing ✅ 
LGTM→ failing ❌ 

*Code coverage reporting on the `master` branch is currently incorrect. This will be fixed in a later PR

Merge after
 - [ ] LGTM fixed
 - [ ] Review from @Quantalabs

**Comment by quantum9Innovation** [comment]
## Update
Waiting for @Quantalabs before fixing LGTM alerts

**Comment by quantum9Innovation** [comment]
> ## Update
> Waiting for @Quantalabs before fixing LGTM alerts

@Quantalabs

File: epispot/__init__.py [hunk 1]
```diff
@@ -1,28 +1,173 @@
...
-These strings are formatted in Github-flavored markdown.
+These strings are formatted in GitHub-flavored markdown.
 Additionally, all files will have a 'STRUCTURE' label.
+
+## Credits
+The epispot package is supported by the following contributors:
+ - Head of Software & Development: [@quantum9innovation](https://www.github.com/quantum9innovation)
+ - Head of Code Maintenance: [@Quantalabs](https://www.github.com/quantalabs)
+
+Thank you also to all [GitHub contributors](https://www.github.com/epispot/epispot/insights/contributors)
```
Comment (Quantalabs) [inline epispot/__init__.py:72]: Why do you have all this? Use:
```rst
.. include:: ../README.md
```

File: epispot/__init__.py [hunk 2]
```diff
@@ -1,28 +1,173 @@
...
+# metadata
+source = 'https://www.github.com/epispot/epispot'
 """URL to GitHub source"""
+docs = 'https://epispot.github.io/epispot'
+"""Project documentation (not version-specific)"""
+issues = 'https://www.github.com/epispot/epispot/issues'
+"""Submit new issues or bugs here"""
+
+
+_sanity_check()  # complete installation
```
Comment (Quantalabs) [inline epispot/__init__.py:173]: You shouldn't have the sanity check after running it.
```suggestion
_sanity_check()  # complete installation
del _sanity_check()
```

File: epispot/__init__.py [hunk 3]
```diff
@@ -1,28 +1,173 @@
...
+                                 >>> conda install matplotlib  # anaconda''')
+
+
 # imports
 import warnings
 import random
+
+
+# dependencies
+_dependency_check()  # check for uninstalled dependencies
```
Comment (Quantalabs) [inline epispot/__init__.py:98]: You shouldn't have the dependency check after running it.
```suggestion
_dependency_check()  # check for uninstalled dependencies
del _dependency_check()
```

**Review by Quantalabs — CHANGES_REQUESTED** [review]

**Comment by quantum9Innovation** [comment]
# Merge Process
 - [ ] CodeQL must pass :heavy_check_mark: 
 - [ ] LGTM must pass :heavy_check_mark: 
 - [ ] Approving review from @Quantalabs :heavy_check_mark: 

---

Merge method undetermined as of yet

**Review by Quantalabs — APPROVED** [review]
Looks good

**Comment by Quantalabs** [comment]
> Looks good

Do fix the LGTM alert however

**Pull request merged**